### PR TITLE
Add external communication explanation

### PR DESCRIPTION
**Pull request by mkorbi**
Updated the communication handbook to reflect the remarks about including release team leads in communication with CNCF.

### CODE REVIEW

**Comment by karenhchu** [comment]
Can we clarify what "release team leads" mean? I think it may be interpreted a couple ways:

- Team Lead + Role Leads
- Team Lead + Team Lead shadows

 Also, which shadows? Specifically the Comms shadows or all shadows?

**Comment by mkorbi** [comment]
@onlydole maybe like "notified" so it is more generic and doesn't bind to mails

@karenhchu yes that needs to be more clear, I would go for " release team role leads and communication shadows"

**Comment by justaugustus** [comment]
@karenhchu @mkorbi -- I've [opened a PR](https://github.com/kubernetes/k8s.io/pull/569) to create `release-comms@kubernetes.io`, which would include:
- 1.18 Release Team Lead & Lead Shadows
- 1.18 Communications Lead & Comms Shadows
- SIG Release Chairs

Let's hold off on doing anything with this PR until that one moves forward.

**Comment by mkorbi** [comment]
looks like release-comms@kubernetes.io is created?

**Comment by justaugustus** [comment]
@mkorbi -- Yep!

From [my email](https://groups.google.com/d/topic/kubernetes-release-team/U84uFHr2L18/discussion):
>Release Team --
>
>In https://github.com/kubernetes/k8s.io/pull/569, I created the release-comms@kubernetes.io Google Group. Moving forward, please use this list for external release communications (communicating with the CNCF, etc.).
>
>The following groups are members:
>- 1.18 Release Team Lead & Lead Shadows
>- 1.18 Communications Lead & Comms Shadows
>- SIG Release Chairs
>
>Max --
>
>Please update https://github.com/kubernetes/sig-release/pull/986 to reference this list in its' own heading above "Slack Channel" (https://github.com/kubernetes/sig-release/pull/986/files#diff-8d8480228a44218d3f2584cc6fd1fc07R33).
>
>The new heading should also include a note that this list needs to be rotated/actively managed every cycle.
>
>-- Stephen

File: release-team/role-handbooks/communications/README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The expected time investment for both leads and shadows are as follows:
 There is a channel on the Kubernetes Slack workspace, `release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations.
 
 ## Release Milestone Activities
+Ensure to have the release team leads and shadows in CC during the communication with CNCF.
```
Comment (onlydole) [inline release-team/role-handbooks/communications/README.md]:
```suggestion
Ensure to have the release team leads and shadows are notified while communicating with the CNCF.
```

Comment (justaugustus) [reply release-team/role-handbooks/communications/README.md]: Please defer reviews until my comment is addressed: https://github.com/kubernetes/sig-release/pull/986#issuecomment-585802554

**Comment by mkorbi** [comment]
@justaugustus @onlydole sorry for the delay, does it fit? :)

File: release-team/role-handbooks/communications/README.md [hunk 2]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
...
+Ensure to have the release team leads and shadows are notified while communicating with the CNCF.
 
 ### Week 1
 
 - Select your shadows
 - Ensure yourself and your shadows are entered into the release contact sheet
 - Ensure yourself and your shadows are on the release team meeting invites
 - Check if there are any holidays or events (KubeCon!) that occur during this release
-  - This can/will affect your communication with the CNCF and SIG Chairs. Plan accordingly
+- This can/will affect your communication with the CNCF and SIG Chairs. Plan accordingly
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]: The previous positioning was correct. This bullet was a note on the previous point about holidays/events.

File: release-team/role-handbooks/communications/README.md [hunk 3]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
...
 
 ### Week 2
 
-- N/A
+- Update the `release-comms@kubernetes.io` Google Group list
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]:
```suggestion
- Update the `release-comms@kubernetes.io` Google Group to ensure the list **_only includes_**:
  - The current release cycle's Release Team Lead & Lead Shadows
  - The current release cycle's Communications Lead & Comms Shadows
  - SIG Release Chairs

  [Membership](https://git.k8s.io/k8s.io/groups/groups.yaml) for this group is defined in [kubernetes/k8s.io](https://git.k8s.io/k8s.io).
```

File: release-team/role-handbooks/communications/README.md [hunk 4]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
...
+* SIG Release Chairs
+
+The list hast to be rotated/actively managed every cycle.
+
 ### Slack Channel
 
 There is a channel on the Kubernetes Slack workspace, `release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations.
 
 ## Release Milestone Activities
+Ensure to have the release team leads and shadows are notified while communicating with the CNCF.
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]: Remove this. It's explained above in the `External Release Communication` section.

File: release-team/role-handbooks/communications/README.md [hunk 5]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
 
 **NOTE:** These are estimates and your personal experience may vary. The more time you can spend working as a team and regularly communicating within your team, the better off your experience will be.
 
+### External Release Communication
+
+Please use the `release-comms@kubernetes.io` Google Group list for external release communications (communicating with the CNCF, etc.).
+
+The following groups are members:
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]:
```suggestion
The following groups should be members:
```

File: release-team/role-handbooks/communications/README.md [hunk 6]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
...
+* 1.18 Release Team Lead & Lead Shadows
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]:
```suggestion
* The current release cycle's Release Team Lead & Lead Shadows
```

File: release-team/role-handbooks/communications/README.md [hunk 7]
```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
...
+* 1.18 Communications Lead & Comms Shadows
```
Comment (justaugustus) [inline release-team/role-handbooks/communications/README.md]:
```suggestion
* The current release cycle's Communications Lead & Comms Shadows
```

File: release-team/role-handbooks/communications/README.md (hunk 4 above)
Comment (mkorbi) [reply release-team/role-handbooks/communications/README.md]: done

File: release-team/role-handbooks/communications/README.md (hunk 2 above)
Comment (mkorbi) [reply release-team/role-handbooks/communications/README.md]: ok

**Comment by justaugustus** [comment]
@mkorbi -- Thanks. The following still needs to be done:

> After they're addressed, I need the following:
> 
> * Commit message and body that describes what is being changed, in which section
> * Squashed commits
> * A PR title that describes what is being changed, in which section

**Comment by justaugustus** [comment]
Looks _much clearer_. Thanks @mkorbi! 
/hold cancel
/kind documentation
/priority important-soon
/lgtm
/approve

**Pull request merged**